src/web/components/SharingPermission/ParticipantsTable.tsx

```diff
@@ -115,3 +121,14 @@ export function ParticipantsTable({
     </table>
   );
 }
+
+export function ParticipantsTable(props: ParticipantsTableProps) {
+  return (
+    <SortableProvider>
+      <ParticipantsTableContent
+        // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
If you merge in `main` you should be able to get rid of this comment - we removed these warnings last week